### ISSUE TITLE
ToTK: Feature Research/Idea: Provide option to change current map ID.

### ISSUE THREAD

**Issue by ApacheThunder** [marcrobledo/savegame-editors#350]
Currently Link's position can be changed but not the map id. So for example you can't set Link to be in one of the shrines or if in a starting save can't put link into the overworld. The special intro scene is on it's own map id seperate from overworld/underworld/shrines.

I'm doing some save file comparison to try and find where the game stores this and ended up finding this:

![image](https://github.com/marcrobledo/savegame-editors/assets/11767416/55a09842-301e-4d9f-bc3e-405aca6ece0a)
This is the bytes that control LInk's rotation. I suppose you could add that to the editor if you feel that is useful. Anyways I'm hoping research is done on this as I may not find this on my own. :P

**Comment by marcrobledo** [comment]
Depths and sky should be the same map as MainField.

I'll try to implement this as soon as possible, thank you!

**Comment by ApacheThunder** [comment]
Yeah I checked and found Sky/Depths to be the same map ID. It's OpeningField and Shrines that will have unique map IDs.
